README.md

```diff
@@ -7,8 +9,69 @@ Note: Install libgit2 with libssh2 support before installing this package.
     $ brew install libgit2 --with-libssh2
 
 
-## Hacking on salt-shaker
+## Quickstart
+    
+Salt shakers requires an initial config file containing the metadata for the local formula. eg,
+
+	formula: my_organisation/local-formula
+
+	dependencies:
+		- some_organisation/test1-formula
+		- another_organisation/testa-formula>=v1.0.0
+		- another_organisation/testb-formula<=v4.0.0
+		- another_organisation/testc-formula==v2.0.0
+		
+To generate and download a list of formula requirements, simply run
+
+	salt-shaker update
+	
+This will also save a list of the requirements and their versions, by default in 'formula-requirements.txt'
+
+If this file exists, you can run 
+
+	salt-shaker refresh
```

Comment:
These commands don't exist - please update them to reflect the name of the commands as they now are